docs/pages/machine-workload-identity/machine-id/deployment/bound-keypair.mdx

```diff
@@ -0,0 +1,174 @@
+---
+title: Deploying Machine ID with Bound Keypair Joining
+description: "How to install and configure Machine ID with Bound Keypair joining"
+---
+
+In this guide, you will install Machine ID's agent, `tbot`, on an arbitrary
+node. This node could be a bare-metal machine, a VM, a container, or any other
+host - the only requirement is that the node has persistent storage.
+
+Note that this is a generic join method. A platform-specific join method may be
+available that is better suited to your environment; refer to the
+[deployment guides](./deployment.mdx) for a full list of options.
+
+## How it works
+
+With Bound Keypair joining, instead of using a static token or relying solely on
+renewable certificates, Machine ID bots generate a unique keypair. Teleport is
+then configured to trust this public key for future joining attempts.
```

Comment:
I would intro it a bit differently. Take it or leave it:

> Bound keypair joining is a more secure and flexible alternative to static token joining.
> Machine ID bots generate a unique keypair, registering the public key with Teleport
> and keeping the private key in their own storage.
>
> Later, when the bot attempts to join the cluster, ...

Reply:
I've tried to iterate on this a bit and rewrote things a bit, the "more secure and flexible alternative" wound up getting promoted up into the intro paragraph